components/server/src/init.ts

```diff
@@ -78,23 +78,33 @@ export async function start(container: Container) {
         }
     });
 
-    const interval = setInterval(async () => {
-        try {
-            const connection = await container.get(TypeORM).getConnection();
-            const pool: any = (connection.driver as any).pool;
-            const activeConnections = pool._allConnections.length;
-            const freeConnections = pool._freeConnections.length;
-
-            pool.on("enqueue", function () {
+    let interval: NodeJS.Timeout;
+
+    try {
+        const connection = await container.get(TypeORM).getConnection();
+        const pool: any = (connection.driver as any).pool;
```

Comment:
This assumes that the pool is never discarded/recreated.

Reply:
I have verified the MySQL drivers don't recreate the pool object. Only if we were closing the TypeOrm connection, which we don't. In case this assumption if wrong we will see error logs and no more metrics, so we can revisit.